configuration/configuration.go

```diff
@@ -131,6 +131,10 @@ type Configuration struct {
 				// Hosts specifies the hosts which are allowed to obtain Let's
 				// Encrypt certificates.
 				Hosts []string `yaml:"hosts,omitempty"`
+
+				// DirectoryURL points to the CA directory endpoint.
+				// If empty, LetsEncrypt is used.
+				DirectoryURL string `yaml:"directoryurl,omitempty"`
```

Comment:
bike-shed/nit-picking; perhaps this should use `directory_url` instead of one word?

Reply:
cc: @lavalleeale do you mind updating this please

Reply:
https://github.com/distribution/distribution/blob/ac79c0d94a03dabb6b57bc76bfe619a25c45bc56/configuration/configuration.go#L16-L17

PLEASE DISREGARD THE ABOVE THREAD; `directoryurl` is the only correct spelling.

Reply:
Thanks @corhere, well spotted. @lavalleeale sorry about our messup. Would you mind reversing to the previous code?